app/views/favourites/favourite_quotes.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @user_favourites.each do |quote| %>
+  <div class="well">
+    <%=  render "quotes/quote", quote: quote %>
```

Comment:
`<%= render quote %>` will look for a `_quote` partial under `quotes/` and will automatically have the object under a `quote` variable.